client/webserver/locales/en-us.go

```diff
@@ -285,8 +285,8 @@ var EnUS = map[string]string{
 	"Pending":                     "Pending",
 	"disable_wallet":              "Disable Wallet",
 	"enable_wallet":               "Enable Wallet",
-	"disable_wallet_warning":      "Note: This wallet will not be connected to when you start the DEX client software and cannot be used until it is enabled.",
-	"enable_wallet_message":       "This wallet will resume operation and might take some time to sync.",
+	"disable_wallet_warning":      "Note: This wallet will not be connected to when you start the DEX client software and cannot be used until it is enabled. This will also disable all token wallets that depend on this wallet.",
+	"enable_wallet_message":       "This wallet will resume operation and might take some time to sync. This will also enable the parent wallet if this is a token wallet.",
```

Comment:
"If this is a token wallet, the wallet for the chain's primary asset will also be enabled."